.github/workflows/build-and-deploy.yml

```diff
@@ -9,23 +9,26 @@ jobs:
     if: ${{ startsWith(github.ref, 'refs/tags/v') }}
     runs-on: ubuntu-latest
     env:
-      image: ghcr.io/opensafely-core/cohortextractor-v2
+      image: ghcr.io/opensafely-core/databuilder
+      ce_image: ghcr.io/opensafely-core/cohortextractor-v2
     steps:
       - uses: actions/checkout@v2
 
-      - name: Get current git tag
-        id: tags
-        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
-
+      # This relies on the tag having a version of the form vX.Y.Z
       - name: Build image
-        run: docker build . --file Dockerfile --tag ${{ env.image }}:latest
+        run: |
+          PATCH="${GITHUB_REF#refs/*/}"
+          MAJOR="${PATCH%%.*}"
+          MINOR="${PATCH%.*}"
+          docker build . --file Dockerfile \
```

Comment:
Maybe add 

```
echo "MAJOR=$MAJOR"
echo "MINOR=$MINOR"
echo "PATCH=$PATCH"
```
so we get clear debugging output in GHA if something goes wrong here?